spec/spec_helper.rb

```diff
@@ -34,7 +36,7 @@
     VirtFS.mount(@ext.recorder, @ext.recording_root)
 
     @root = @ext.mount_point
-    block_dev = VirtFS::BlockIO.new(VirtDisk::BlockFile.new(@ext.path))
+    block_dev = VirtFS::BlockIO.new(VirtDisk::FileIo.new(@ext.path))
```

Comment:
@movitto 
This should be:
```ruby
block_dev = VirtDisk::Disk.new(VirtDisk::FileIo.new(@ext.path))
```
If the disk has partitions, the following will return an array of partitions on the disk:
```ruby
VirtDisk::PartitionType.partition_probe(blovk_dev)
```